src/main/resources/db/migration/changes/Jobs.json

```
@@ -0,0 +1,74 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "Jobs-1",
+          "author": "matt (generated)",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "JOBS"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "CONSTRAINT_2"
```

Do we absolutely need to include a `primaryKeyName`? If so, could we adopt a clearer naming pattern, maybe something like:

```suggestion
                        "primaryKeyName": "PRIMARY_KEY_JOBS"
```